connect/runtime/src/test/java/org/apache/kafka/connect/integration/MonitorableSinkConnector.java

```diff
@@ -60,7 +60,7 @@ public void start(Map<String, String> props) {
         @Override
         public void put(Collection<SinkRecord> records) {
             super.put(records);
-            for (SinkRecord ignore : records) {
+            for (@SuppressWarnings("UnusedLocalVariable") SinkRecord ignore : records) {
```

Comment:
Since `records` is a Collection, can we just do `count += records.size()` ?

Reply:
Done